### ISSUE TITLE
Illustrated books without style

### ISSUE THREAD

**Issue by polarbearing** [NiklasGollenstede/epub-creator#4]
Thanks for the effort! 
I tried in v0.5.4b8 (style setting = on)  to convert some illustrated books, that typically have a large image all over the page, and the text rendering highly depending on the original font, being placed onto the appropriate places, and font settings such as colour (white font over dark illustration). 

In the converted ebook, the images are in the middle, but the text is unstyled in the left upper corner of the page.

**Comment by NiklasGollenstede** [comment]
Is this with a book in `overdrive.com` or with some other website?

I should have added this to the "Keep style information" setting before:
> Only applies to <code>overdrive.com</code>, reader mode always removes all styles.

If it is on `overdrive.com`, could you please send me a link to a (publicly accessible sample of a) book where this happens, and name a reader software this is supposed to work with?

**Comment by polarbearing** [comment]
Examples:
https://www.overdrive.com/media/1744309/star-wars-sith-wars
https://www.overdrive.com/media/2557260/the-third-wheel

In the second type of book, probably character set information is lost also, which appears in an English text as strange characters instead of curly quotes/apostrophes. 
Generated epub tested in Calibre and ADE.

**Comment by polarbearing** [comment]
As these types of books are very page-oriented, conversion into PDFs would be an alternative.

**Comment by NiklasGollenstede** [comment]
Ok, I see the problem now. There was a simple error in the code which I just fixed, but that won't be enough to support this kind of layout. With this kind of book, the dimensions of the page seem completely fixed, and the surrounding viewer scales the entire page to fit your screen. That is not how HTML (which is what you have inside of ePubs) is supposed to be used, and I kind of doubt it would work in other viewers even with a perfect copy of the page as displayed on overdrive.com.

So I am not sure whether I even want to support this kind of books. The ePub format is meant for text based books that can reflow based on the current screen size and font settings.
For fixed layouts, PDF seems much more appropriate as a format, but generating PDFs from generic web pages is not really possible.

So bottom line: this isn't supported, I don't think I will support it, and if then not soon. Sorry!

**Comment by polarbearing** [comment]
Thanks for the first fix, let me know when committed. 

I agree that PDF is better suited for such layout-dependant books. 
Interestingly, however, the first few books on the Wimpy series is natively available as epub on overdrive, only the more recent just on kindle and read-in-browser. E.g.:
https://www.overdrive.com/media/4428593/diary-of-a-wimpy-kid

**Comment by NiklasGollenstede** [comment]
Ok, I committed that fix, but I don't think it will help much. Depending on the reader you use the text might now be positioned as intended, but since font information are still missing, that is not necessarily an improvement.

> Interestingly, however, the first few books on the Wimpy series is natively available as epub on overdrive

Yes, but in the first example from that series you linked there are text clipping errors even on overdrive. I don't think anything else needs to be said about that :D
